swig/include/python/gdal_python.i

```diff
@@ -4939,3 +4939,26 @@ def quiet_errors():
         tuple.max_part_count = max_part_count
         val = tuple
 %}
+
+%feature("shadow") InterpolateAtPoint %{
+def InterpolateAtPoint(self, *args, **kwargs):
+    """InterpolateAtPoint(Band self, pixel, line, interpolation) -> value
+
+    ... doc here TODO ...
```

Comment:
I'm not sure which of this inline function or gdal_band_docs.i wins over the other one, but if this is this one, it should be updated. Otherwise removed

Reply:
done. This one wins over.